src/elements/hoverboard-analytics.html

```diff
@@ -1,28 +1,56 @@
 <script>
 
+
   class HoverboardAnalytics extends HTMLElement {
+    
     connectedCallback() {
-      // track metrics with google analytics...
+
+      // GOOGLE ANALYTICS TRACKING
+
+      // Load google analytis script
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
       (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
       m=s.getElementsByTagName('body')[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      // Load autotrack script
+      (function(s,o,g,a,m){a=s.createElement(o),m=s.getElementsByTagName('body')[0];
+      a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})
+      (document,'script','../../node_modules/autotrack/autotrack.js');
```

Comment:
add to `polymer.json` into `extraDependencies`
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/polymer.json#L24